ext/opcache/shared_alloc_mmap.c

```diff
@@ -39,7 +39,11 @@
 static int create_segments(size_t requested_size, zend_shared_segment ***shared_segments_p, int *shared_segments_count, char **error_in)
 {
 	zend_shared_segment *shared_segment;
+                                          int flags = PROT_READ | PROT_WRITE;
```

Comment:
Something went wrong with the indentation here...

Reply:
oups